boot/bootdata/hivesys.inf

```diff
@@ -5,7 +5,7 @@ Signature = "$Windows NT$"
 
 HKLM,"SYSTEM\CurrentControlSet\Control","CurrentUser",2,"USERNAME"
 HKLM,"SYSTEM\CurrentControlSet\Control","WaitToKillServiceTimeout",2,"20000"
-HKLM,"SYSTEM\CurrentControlSet\Control\Biosinfo","InfName",2,"biosinfo.inf"
+;HKLM,"SYSTEM\CurrentControlSet\Control\Biosinfo","InfName",2,"biosinfo.inf"
```

Comment:
```suggestion
; HKLM,"SYSTEM\CurrentControlSet\Control\Biosinfo","InfName",2,"biosinfo.inf" ; UNIMPLEMENTED
```

Reply:
You won't start adding UNIMPLEMENTED to all commented lines in all .inf files?

Reply:
> `git blame`

(It's a pain, especially as https://git.reactos.org/ does not support it.) Anyway.